CRM/Upgrade/Incremental/php/FourThree.php

```diff
@@ -1188,19 +1188,6 @@ public function replaceContributionTypeId(CRM_Queue_TaskContext $ctx, $query, $t
             }
           }
         }
-        foreach (['select_tables', 'where_tables'] as $value) {
-          if (preg_match('/contribution_type/', $dao->$value)) {
-            $tempValue = unserialize($dao->$value);
-            if (array_key_exists('civicrm_contribution_type', $tempValue)) {
-              $tempValue['civicrm_financial_type'] = $tempValue['civicrm_contribution_type'];
-              unset($tempValue['civicrm_contribution_type']);
-            }
-            $saveDao->$value = serialize($tempValue);
-          }
-        }
-        if (preg_match('/contribution_type/', $dao->where_clause)) {
-          $saveDao->where_clause = preg_replace('/contribution_type/', 'financial_type', $dao->where_clause);
-        }
```

Comment:
At what point did the columns become unused? Were they migrated to something else?

The reason I ask - it seems like this might affect the correctness of a DB upgrade that passes through v4.3?

Maybe that's OK. Maybe we could get away with bumping `MINIMUM_UPGRADABLE_VERSION` from `4.2` to `4.6`?

Reply:
@totten looking at https://github.com/civicrm/civicrm-svn/search?p=1&q=where_clause&unscoped_q=where_clause It seems as tho it hasn't really been used at all.